README.md

```diff
@@ -37,10 +47,12 @@ Each intention can appear at any of the 4 risk levels. Each intention's full det
 | --- | --- | --- |
 | F | Feature | Change or extend one aspect of program behavior without altering others. |
 | B | Bugfix | Repair one existing, undesirable program behavior without altering any others. |
-| R | Refactoring | Change implementation without changing program behavior. |
-| D | Documentation | Change something which communicates to team members and does not impact program behavior. |
+| r | Refactoring | Change implementation without changing program behavior. |
+| d | Documentation | Change something which communicates to team members and does not impact program behavior. |
```

Comment:
Capitalization is inconsistent.  Is this intentional?
- Yes -- Explain why.
- No -- Make it consistent.

Reply:
Uppercase = intended to change behavior.

Lowercase = intended to preserve behavior.

Reply:
Ah... understood.

Could this be added as a key to the table?


UPPERCASE = intended to change behavior
lowercase = intended to preserve behavior

Reply:
That was the intention, and now I'm thinking to go a different way: allow F or f for feature, with F meaning this is user-visible, and f meaning it changes user-invisible functionality. Can be used to decide what should appear in release notes / changelog. I'm going to provide for 2 variations here, by letting the team choose the meaning of capitalization.